plugins/catalog-backend-module-incremental-ingestion/src/module/WrapperProviders.ts

```diff
@@ -130,6 +132,18 @@ export class WrapperProviders {
         frequency,
         timeout: length,
       });
+
+      const topics = engine.supportsEventTopics();
+      if (topics.length > 0) {
+        logger.info(
+          `Provider ${provider.getProviderName()} subscribing to events for topics: ${topics.join()}`,
+        );
+        await this.options.events.subscribe({
+          topics,
+          id: provider.getProviderName(),
```

Comment:
is this id unique enough or will it need a prefix like `catalog-backend-module-incremental-ingestion:` before it or so

Reply:
I don't know to be honest - it just says in the tsdoc that it's used for logging purposes.

Reply:
https://github.com/backstage/backstage/pull/27766 😉